### PR TITLE
Polished grammar, spelling, and formatting of Documentation

### PR DESCRIPTION
**Pull request by Blake192**

### CODE REVIEW

File: docs/getting-started/installation/bare-metal.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
 order: 100
 ---
-# Self-hosted server
+# Bare-metal
```
Comment (Bubka) [inline docs/getting-started/installation/bare-metal.md]: Please restore the Self-hosted term. Bare-metal is too restrictive whereas Self-hosted is commonly used.

**Review by Bubka — CHANGES_REQUESTED** [review]
Great review, thank you very much! My English still has to be improved.

Please just restore _Self-hosted_ in place of _Bare-metal_ (see related comment).

File: docs/getting-started/installation/bare-metal.md (hunk 1 above)
Comment (Blake192) [reply docs/getting-started/installation/bare-metal.md]: Fixed with latest commit

**Comment by Blake192** [comment]
Another thing I changed in this PR was giving the `docker-compose.yml` file its own section to allow people to dive into the project faster (kind trying to mimic how linuxserver.io does it). It would be smart to assign a github action to update the Docker-Compose section of the documentation with the file every time the file is changed for efficiency.

**Review by Bubka — APPROVED** [review]

**Pull request merged**